pkgs/tools/admin/cdist/default.nix

```
@@ -0,0 +1,51 @@
+{ lib, buildPythonApplication, fetchFromGitea, sphinxHook, sphinx-rtd-theme }:
```

One item per line make the diffs easier to read in the future.

```suggestion
{ lib
, buildPythonApplication
, fetchFromGitea
, sphinxHook
, sphinx-rtd-theme
}:
```